plugins/basic/sg_client.py

```diff
@@ -34,77 +19,52 @@ def log(msg):
 
     print(message)
 
-# Keep a reference on the connection
-_connection = None
-
 # We only initialize Shotgun once in the client process
 _shotgun_is_initialized = False
 
-# The Shotgun service class and its instance
-_service = None
-
-# Number of attempts to reconnect to the server
-_nb_connection_attempts = 100
-
 ### Unity API access
 def GetUnityEngine():
```

Comment:
This method and the other one can be removed

Reply:
A lot of calls are being made to this method.removing them makes bootstrap fail

Reply:
You should be able to search all the files for `GetUnityEngine()` and then put `import UnityEngine` at the top of the file and replace the function call with just `UnityEngine`.

Reply:
Same thing with GetUnityEditor()